docs/ref/clvm.md

```diff
@@ -603,15 +603,139 @@ When used in an integer context, nil behaves as zero.
+
+We'll start with a table showing the two costs that must be applied to all operators, namely a mandatory cost and a cost per byte of memory.
+
+| type                   | base cost   | cost per byte |
+| ---------------------- | ----------- | ------------- |
+| mandatory cost         | 1           | -             |
+| `MALLOC_COST_PER_BYTE` | -           | 10            |
+
+The following table shows the cost of each CLVM operator, as well as the cost of the outputted conditions.
+
+| operator            | base cost | cost per arg | cost per byte |
+| ------------------- | --------- | ------------ | ------------- |
+| `f` *first*         | 30        | -            | -             |
+| `i` *if*            | 33        | -            | -             |
+| `c` *cons*          | 50        | -            | -             |
+| `r` *rest*          | 30        | -            | -             |
+| `l` *listp*         | 19        | -            | -             |
+| `q` *quote*         | 20        | -            | -             |
+| `a` *apply*         | 90        | -            | -             |
+| `=`                 | 117       | -            | 1             |
+| `+`                 | 99        | 320          | 3             |
+| `/`                 | 988       | -            | 4             |
+| `*`                 | 92        | 885          | [see here](https://github.com/Chia-Network/clvm_tools/blob/main/costs/README.md#multiplication) |
+| `logand`, `logior`, `logxor` | 100 | 264       | 3             |
+| `lognot`            | 331       | -            | 3             |
+| `>`                 | 498       | -            | 2             |
+| `>s`                | 117       | -            | 1             |
+| `strlen`            | 173       | -            | 1             |
+| `concat`            | 142       | 135          | 3             |
+| `divmod`            | 1116      | -            | 6             |
+| `sha256`            | 87        | 134          | 2             |
+| `ash`               | 596       | -            | 3             |
+| `lsh`               | 277       | -            | 3             |
+| `not`, `any`, `all` | 200       | 300          | -             |
+| `point_add`         | 101094    | 1343980      | -             |
+| `pubkey_for_exp`    | 1325730   | -            | 38            |
+|                     |           |              |               |
+
+Finally, three of CLVM's conditions also have an associated cost:
+
+| operator            | cost    |
+| ------------------- | ------- |
+| `CREATE_COIN`       | 1800000 |
+| `AGG_SIG_UNSAFE`    | 1200000 |
+| `AGG_SIG_ME`        | 1200000 |
+
+Aside from cost, the maximum number of atoms or pairs in a CLVM program is 2^31. If this threshold is exceeded, the program will fail.
+
+## Evaluating cost for a sample brun program
+
+In this section, we'll show you how to calculate the cost of a simple CLVM program by hand. The program we'll use is `brun (concat (q . fu) (q . bar))`.
```

Comment:
it bothers me a bit that we allow atoms like this without quotes. I don't know if I'm the only one, but I would prefer to quote `fu` and `bar`, to make it more clear what those are.

Reply:
I changed it to the way you suggested. I also changed the word because the original is probably not a great word to have in our official documentation.